subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -106,6 +106,7 @@ static int download_client_callback(const struct download_client_evt *event)
 				/* Abort current download procedure, and
 				 * schedule new download from offset.
 				 */
+				(void) download_client_disconnect(&dlc);
```

Comment:
nit: no need for a space after `(void)` cast

Reply:
Fixed.